src/many-migration/src/lib.rs

```diff
@@ -556,7 +556,6 @@ impl<'a, T, E, IDX: AsRef<str>> Index<IDX> for MigrationSet<'a, T, E> {
 }
 
 /// Kept for backward compatibility.
-#[deprecated = "Should use MigrationSet::load() instead."]
 pub fn load_migrations<'a, T, E>(
```

Comment:
Can you add a comment to NOT use those?

Reply:
@fmorency Yeah . . . will re-add as a simple comment

Reply:
Can you explain here why you want to remove the deprecation warning? This method should not be used anymore.